web/src/lib/components/admin-page/settings/library-settings/library-settings.svelte

```diff
@@ -60,25 +61,13 @@
               bind:checked={config.library.scan.enabled}
             />
 
-            <div class="flex flex-col my-2 dark:text-immich-dark-fg">
-              <label
-                class="font-medium text-immich-primary dark:text-immich-dark-primary text-sm"
-                for="expression-select"
-              >
-                {$t('admin.cron_expression_presets')}
-              </label>
-              <select
-                class="p-2 mt-2 text-sm rounded-lg bg-slate-200 hover:cursor-pointer dark:bg-gray-600"
-                disabled={disabled || !config.library.scan.enabled}
-                name="expression"
-                id="expression-select"
-                bind:value={config.library.scan.cronExpression}
-              >
-                {#each cronExpressionOptions as { title, expression }}
-                  <option value={expression}>{title}</option>
-                {/each}
-              </select>
-            </div>
+            <SettingSelect
+              options={cronExpressionOptions}
+              disabled={disabled || !config.library.scan.cronExpression}
```

Comment:
Is it intentional that you changed this from `enabled` to `cronExpression`?

Reply:
Oops, copy paste error :sweat_smile: Fixed